regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -80,3 +80,9 @@ <h3 class="toc-type">Table of Contents</h3>
 {% endblock %}
 
 {% block chrome-sidebar %}{% endblock %}
+
+{% block preload %}
+<script>
+  window.DOC_NUMBER = '{{ doc_number }}';
+</script>
```

Comment:
A related question: do we want users to be able to comment on multiple proposals without changing URLs? My assumption has been that we don't, but that can change. If that's correct, then the proposal number is effectively a global setting--we can put it on a DOM element (e.g. as `data-doc-number`) instead, but it's still global, and the JS has to reach into the DOM to instantiate the collection.